### PR TITLE
feat(route): 支持 xsijishe.com 论坛 rss

### PR DESCRIPTION
**Pull request by akynazh**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
/xsijishe/forum/51
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [x] 全文获取 fulltext
  - [x] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: docs/bbs.md [hunk 1]
```diff
@@ -869,6 +869,20 @@ pageClass: routes
 
 <Route author="nczitzk" example="/newsmth/account/fef705ec94819a5a87941759e33c0982" path="/newsmth/account/:id" :paramsDesc="['用户 id，可在用户页的 URL 中找到']"/>
 
+## 司机社
+
+### 论坛
+
+<Route author="akynazh" example="/sjs/forum/51" path="/sjs/forum/:fid" :paramDesc="['子论坛 id']">
```
Comment (TonyRL) [inline docs/bbs.md:876]: Please update path to `/xsijishe/forum/:fid`

Comment (akynazh) [reply docs/bbs.md]: done

File: docs/bbs.md [hunk 2]
```diff
@@ -869,6 +869,20 @@ pageClass: routes
 
 <Route author="nczitzk" example="/newsmth/account/fef705ec94819a5a87941759e33c0982" path="/newsmth/account/:id" :paramsDesc="['用户 id，可在用户页的 URL 中找到']"/>
 
+## 司机社
+
+### 论坛
+
+<Route author="akynazh" example="/xsijishe/forum/51" path="/xsijishe/forum/:fid" :paramDesc="['子论坛 id']">
```
Comment (TonyRL) [inline docs/bbs.md:876]:
```suggestion
<Route author="akynazh" example="/xsijishe/forum/51" path="/xsijishe/forum/:fid" :paramDesc="['子论坛 id']" radar="1">
```

File: docs/bbs.md [hunk 3]
```diff
@@ -869,6 +869,20 @@ pageClass: routes
...
+
+::: 关于子论坛 id 的获取方法
```
Comment (TonyRL) [inline docs/bbs.md:878]:
```suggestion
::: tip 关于子论坛 id 的获取方法
```

File: lib/v2/xsijishe/forum.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    const $ = cheerio.load(resp.data);
+    const forumCategory = $('.nex_bkinterls_top .nex_bkinterls_ls a').text();
+    let items = $('.nex_forum_lists')
+        .toArray()
+        .filter((item) => {
+            // remove #stickthread
+            item = $(item);
+            return item.attr('id').includes('normalthread');
+        })
+        .map((item) => {
```
Comment (TonyRL) [inline lib/v2/xsijishe/forum.js:19]:
```suggestion
    let items = $('[id^="normalthread"]')
        .toArray()
        .map((item) => {
```

File: lib/v2/xsijishe/forum.js [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+            item = $(item);
+            const nexAuthorBtms = item.find('.nex_author_btms');
+            const nexForumtitTopA = item.find('.nex_forumtit_top a');
```
Comment (TonyRL) [inline lib/v2/xsijishe/forum.js:22]:
```suggestion
            const nexForumtitTopA = item.find('.nex_forumtit_top a').first();
```

File: lib/v2/xsijishe/forum.js [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+            };
+        });
+    items = await Promise.all(
+        items.map((item) =>
+            ctx.cache.tryGet(item.link, async () => {
+                const resp = await got(item.link);
+                const $ = cheerio.load(resp.data);
+                const firstViewBox = $('.viewbox').first();
+                firstViewBox.remove('script, .allrepliestit, #reply_order_menu');
+                item.description = firstViewBox.html();
```
Comment (TonyRL) [inline lib/v2/xsijishe/forum.js:45]: images are lazy-loaded, you need to fix them
```suggestion
                const firstViewBox = $('.t_f').first();

                firstViewBox.find('img').each((_, img) => {
                    img = $(img);
                    if (img.attr('zoomfile')) {
                        img.attr('src', img.attr('zoomfile'));
                        img.removeAttr('zoomfile');
                        img.removeAttr('file');
                    }
                    img.removeAttr('onmouseover');
                });

                item.description = firstViewBox.html();
```

File: lib/v2/xsijishe/radar.js [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+            {
+                title: '司机社论坛',
+                docs: 'https://docs.rsshub.app/bbs.html#si-ji-she',
+                source: ['/*'],
+                target: (_, url) => {
+                    const re = /forum-[0-9]+/;
+                    const res = re.exec(url);
+                    if (res) {
+                        return `/xsijishe/forum/${res[0].replace('forum-', '')}`;
+                    }
```
Comment (TonyRL) [inline lib/v2/xsijishe/radar.js:14]:
```suggestion
                title: '论坛',
                docs: 'https://docs.rsshub.app/bbs.html#si-ji-she',
                source: ['/*'],
                target: (_, url) => {
                    const re = /forum-(\d+)-/;
                    const res = re.exec(url);
                    if (res) {
                        return `/xsijishe/forum/${res[1]}`;
                    }
```

**Pull request merged**